docs/developers/keystore.md

```diff
@@ -0,0 +1,56 @@
+---
+id: keystore
+title: MultiversX Smart Contracts
+---
+
+[comment]: # (mx-abstract)
+
+A keystore is a mechanism for storing private keys. It is a JSON file that encrypts a private key and is the standard for interchanging keys between devices as until a user provides their password, their key is safe.
+
+Before continuing, please read this reference: [ERC-2335: BLS12-381 Keystore](https://eips.ethereum.org/EIPS/eip-2335).
```

Comment:
This can be dropped below as merely an auxiliary reference, I think. Not very important for developers to know about it.

Reply:
moved at the end of the page